lib/ui/page/home/page/user/view.dart

```diff
@@ -349,11 +347,50 @@ class UserView extends StatelessWidget {
     // final bool favorite =
     //     c.contact.value?.contact.value.favoritePosition != null;
 
+    final bool favorite =
+        c.user?.dialog.value?.chat.value.favoritePosition != null;
+
+    final bool muted = c.user?.dialog.value?.chat.value.muted != null;
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
         const SizedBox(height: 8),
+        ActionButton(
+          key: Key(favorite ? 'UnfavoriteChatButton' : 'FavoriteChatButton'),
+          text: favorite
+              ? 'btn_delete_from_favorites'.l10n
+              : 'btn_add_to_favorites'.l10n,
+          trailing: SvgIcon(
+            favorite ? SvgIcons.favoriteSmall : SvgIcons.unfavoriteSmall,
+          ),
+          onPressed: favorite ? c.unfavoriteChat : c.favoriteChat,
+        ),
 
+        ActionButton(
+          key: Key(muted ? 'UnmuteChatButton' : 'MuteChatButton'),
+          text: muted
+              ? PlatformUtils.isMobile
+                    ? 'btn_unmute'.l10n
+                    : 'btn_unmute_chat'.l10n
+              : PlatformUtils.isMobile
+              ? 'btn_mute'.l10n
+              : 'btn_mute_chat'.l10n,
+          trailing: SvgIcon(muted ? SvgIcons.unmuteSmall : SvgIcons.muteSmall),
+
+          onPressed: muted ? c.unmuteChat : c.muteChat,
+        ),
+        ActionButton(
+          key: const Key('ClearHistoryButton'),
+          text: 'btn_clear_chat'.l10n,
+          trailing: const SvgIcon(SvgIcons.cleanHistory),
+          onPressed: () => _clearChat(c, context),
+        ),
+        ActionButton(
+          key: const Key('HideChatButton'),
+          text: 'btn_delete_chat'.l10n,
+          trailing: const SvgIcon(SvgIcons.delete),
```

Comment:
Those icons look a bit disproportional, don't you think?

<img width="502" alt="Screenshot 2025-05-27 at 18 15 18" src="https://github.com/user-attachments/assets/c612cea6-442b-43d3-8782-7326241c4e36" />